src/main/java/team/themoment/hellogsmv3/domain/member/service/QueryMemberAuthInfoByIdService.java

```diff
@@ -0,0 +1,29 @@
+package team.themoment.hellogsmv3.domain.member.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import team.themoment.hellogsmv3.domain.member.dto.response.FoundMemberAuthInfoResDto;
+import team.themoment.hellogsmv3.domain.member.entity.Member;
+import team.themoment.hellogsmv3.domain.member.repo.MemberRepository;
+import team.themoment.hellogsmv3.global.exception.error.ExpectedException;
+
+@Service
+@RequiredArgsConstructor
+public class QueryMemberAuthInfoByIdService {
+
+    private final MemberRepository memberRepository;
+
+    public FoundMemberAuthInfoResDto execute(Long memberId) {
+        Member member = memberRepository.findById(memberId)
+                .orElseThrow(() -> new ExpectedException("존재하지 않는 지원자입니다. member ID: " + memberId, HttpStatus.NOT_FOUND));
+
+        return FoundMemberAuthInfoResDto
+                .builder()
+                .memberId(member.getId())
+                .email(member.getEmail())
+                .authReferrerType(member.getAuthReferrerType())
+                .role(member.getRole())
+                .build();
+    }
```

Comment:
```suggestion
        return FoundMemberAuthInfoResDto.builder()
                .memberId(member.getId())
                .email(member.getEmail())
                .authReferrerType(member.getAuthReferrerType())
                .role(member.getRole())
                .build();
    }
```

위와 같은 format이 더 가독성이 뛰어나다고 생각합니다.
현재 다른 코드들도 위와 같은 format으로 되어있습니다.

Reply:
반영하였습니다
[b12621b](https://github.com/themoment-team/hellogsm-server-24/pull/60/commits/b12621b1f2d4910f1f768b9c1cc619af28e343a8)